sops/__init__.py

```diff
@@ -95,6 +95,8 @@
 
 INPUT_VERSION = VERSION
 
+DEFAULT_UNENCRYPTED_SUFFIX = '_unencrypted'
```

Comment:
What about just `UNENCRYPTED_SUFFIX` ? We might modify it in `initialize_tree`, so it may not always be a default.